spec/models/event_spec.rb

```diff
@@ -0,0 +1,55 @@
+require "rails_helper.rb"
+
+RSpec.describe Event do
+  it { is_expected.to have_db_column(:title) }
+  it { is_expected.to have_db_column(:starts_at) }
+  it { is_expected.to have_db_column(:description) }
+
+  it { is_expected.to have_db_index(:user_id) }
+  it { is_expected.to belong_to(:user) }
+
+  it { is_expected.to have_db_index(:place_id) }
+  it { is_expected.to belong_to(:place) }
+
+  describe "predicate methods" do
+    context "event's starting time was in the past" do
+      before(:each) do
```

Comment:
I also prefer to write one line before filter like `before { . . . }`, but again this is fine too.